### PR TITLE
[IT-Translation] PSA header and files

### PR DESCRIPTION
**Pull request by hrumag**
Dear all,
let's start the review for Italian translation of PSA Outreach Docs.

Just some notes below:

- I worked on .sla files only -> no .pdf file has been generated yet (I talked with @lh1008 who explained me the workflow of the graphical aspect of the .pdf files) -> the .pdf files will be added in a separate PR by the Outreach Team, if you agree.
- I noticed a footnote on every file, "**Monero means Money**". I would translate it as "**Monero significa Denaro**". I did not found any text entry for this, so I would handle it here.
- Commit squashes will occur at the end of the review.

@erciccione @serhack, your turn. :smile:

### CODE REVIEW

**Comment by serhack** [comment]
Ops, I forgot about this.

File: monero-outreach-docs/translations/it/PSA/headers/PSA _it_headers.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Page 5
+ - Monero Teamwork
+ - Lavoro di gruppo Monero
+
+-from start to finish!
+-dall'inizio alla fine!
+
+Page 6
+ - How about we go back to my place and you give me your xmr private keys?
+ - Che ne dici se andiamo a casa mia e mi dai le tue chiavi xmr private?
```
Comment (serhack) [inline monero-outreach-docs/translations/it/PSA/headers/PSA _it_headers.txt]: 😆

**Comment by hrumag** [comment]
Any other contribution?

**Comment by lh1008** [comment]
@hrumag I think you can squash so we can go ahead and merge it. :)

**Pull request merged**